PlatformTools/Tools/bootstrapper/mfs/genoa_poc.yml

```diff
@@ -29,6 +29,9 @@ repositories:
       - action: copy
         source: Platform/PlatformTools/root_dbuild.cmd
         destination: dbuild.cmd
+      - action: copy
+        source: Platform/PlatformTools/root_dbuild.sh
+        destination: dbuild.sh

```

Comment:
Can you make dbuild.sh executable?

Reply:
done https://github.com/openSIL/EDKII-Platform/pull/4/commits/a4e5af4fa917c42c33382ee6f23a15a69b88ec68